Docs/DevSetup/k8s-private-registry.md

```diff
@@ -1,16 +1,20 @@
 # How To Connect Local Kubernetes with our Private Container Registry
 
-Instructions on setting up Kubernetes with a private registry are [here](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/)
-Instructions on authenticating to Azure Container Registry are [here](https://docs.microsoft.com/en-us/azure/container-registry/container-registry-authentication) 
-
 Eventually, a service principle will be set up. In the interim, the best approach is to create a local secret called regcredlocal using the command line as below.
 
-```bash
-kubectl create secret docker-registry regcredlocal --docker-server=gpitfuturesdevacr.azurecr.io --docker-username=gpitfuturesdevacr --docker-password=<password> --docker-email=<your-hscic.gov.uk email> --namespace buyingcatalogue
-```
 
-### To retrieve the relevant password:
+### To retrieve the 'docker password':
 You will need to look in the azure portal http://portal.azure.com
 Log in using your hscic.gov.uk email adress.
 
 Go to: All services > Containers > Container registries > gpitfuturesdevacr > Access Keys > password
```

Comment:
Very minor but from the home page of http://portal.azure.com I couldn't find the All services link. @assoomro suggested changing the link to 'https://portal.azure.com/#allservices' & then remove the 'All services' from the Go to text.

Reply:
I've amended the link to take you to container registries